Leaf API.postman_environment.json

```diff
@@ -186,9 +186,39 @@
 			"key": "trimble_application_name",
 			"value": "fill with your app's",
 			"enabled": true
+		},
+		{
+			"key": "raven_client_id",
+			"value": "fill with your app's",
+			"enabled": true
+		},
+		{
+			"key": "raven_client_secret",
+			"value": "fill with your app's",
+			"enabled": true
+		},
+		{
+			"key": "raven_redirect_url",
+			"value": "fill with your app's",
+			"enabled": true
+		},
+		{
+			"key": "raven_code",
+			"value": "code after redirecting user",
+			"enabled": true
+		},
+		{
+			"key": "raven_refresh_token",
+			"value": "(auto)",
+			"enabled": true
+		},
+		{
+			"key": "leaf_auth_raven",
+			"value": "raven-oauth2-helper.withleaf.io",
+			"enabled": true
```

Comment:
This property should follow the pattern for the other leaf auth helpers, which is to include the property as a postman collection property.

Reply:
Done.